pkg/logger/server.go

```diff
@@ -35,9 +35,11 @@ type LoggerHandler struct {
 	sourceUri *url.URL
 	logMode   v1alpha2.LoggerMode
 	modelId   string
+	namespace string
+	endpoint  string
 }
 
-func New(log logr.Logger, svcHost string, svcPort string, logUrl *url.URL, sourceUri *url.URL, logMode v1alpha2.LoggerMode, modelId string) http.Handler {
+func New(log logr.Logger, svcHost string, svcPort string, logUrl *url.URL, sourceUri *url.URL, logMode v1alpha2.LoggerMode, modelId string, namespace string, endpoint string) http.Handler {
```

Comment:
I know this isn't exactly your code :). While you're in here, do you mind refactoring modelId to inferenceService or isvc? The concepts are slightly different, and it can be confusing if we're not crisp about them.

Reply:
Yeah it is the InferenceService name so it should be called that. Have made the change.